Fitness/Controllers/HomeViewController+Extensions.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  HomeViewController+Extensions.swift
+//  Fitness
+//
+//  Created by Kevin Chan on 5/20/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+extension HomeViewController: UICollectionViewDataSource {
+
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return sections.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 1
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let sectionType = sections[indexPath.section]
+        switch sectionType {
+        case .checkIns:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.checkInsListCellIdentifier, for: indexPath) as! CheckInsListCell
+            cell.configure(for: habits)
+            return cell
+        case .allGyms:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.gymsListCellIdentifier, for: indexPath) as! GymsListCell
+            cell.delegate = self
+            cell.configure(for: self.favoriteGyms)
+            return cell
+        case .todaysClasses:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.todaysClassesListCellIdentifier, for: indexPath) as! TodaysClassesListCell
+            cell.delegate = self
+            cell.configure(for: gymClassInstances)
+            return cell
+        default:
+            // swiftlint:disable:next force_cast
```

Comment:
nitpick, but probably cleaner to move all of these `swiftlint` comments onto the same line as the case
statement
Is there no way to add this kind of case to swiftlint though? I'm sure it's common everywhere

Reply:
we want to keep it as a rule b/c for most cases we shouldn't be force casting